samples/openmp_testsuite/Makefile

```diff
@@ -0,0 +1,37 @@
+TOP=$(abspath ../..)
+include $(TOP)/defs.mak
+
+CONTAINER_NAME=omp.myst.apline.062021
+
+ifdef STRACE
+OPTS = --strace
+endif
+
+OPTS += --memory-size=256m
+
+APPDIR=$(CURDIR)/appdir
+
+ROOTFS=$(CURDIR)/rootfs
+
+TEST_FILE = passed.tests
+
+all: $(APPDIR) $(ROOTFS)
+
+$(ROOTFS):
+	sudo $(MYST) mkcpio $(APPDIR) $(ROOTFS)
+
+$(APPDIR):
+	myst-appbuilder Dockerfile
+
+TESTS=$(shell cat $(TEST_FILE))
+
+define NL
+
```

Comment:
What's line 28-31 for?

Reply:
It adds a new line so this:
![image](https://user-images.githubusercontent.com/9376183/123478586-7e9d4e00-d5b4-11eb-8b90-bef33ab34920.png)
becomes this:
![image](https://user-images.githubusercontent.com/9376183/123478650-907ef100-d5b4-11eb-97cc-85ff9d3a9f05.png)